tldw_Server_API/app/api/v1/schemas/archetype_schemas.py

```
@@ -0,0 +1,29 @@
+    icon: str | None = None
+    persona: ArchetypePersona | None = None
+
+
+class ArchetypeSummary(BaseModel):
+    key: str
+    label: str
+    tagline: str | None = None
+    icon: str | None = None
+
+
+__all__ = ["ArchetypePersona", "ArchetypeTemplate", "ArchetypeSummary"]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)